tpl/barrierless_prod_mess.tpl

```diff
@@ -0,0 +1,45 @@
+!****************************************
+    Bimolecular     {bimol_n} ! {products} 
+
+    Fragment       {fr_n} ! {chemid fragment 1}
+    RRHO
+        Geometry[angstrom]            {n_atoms frag 1}
+        {geom fragment 1}
+
+        Core   RigidRotor
+            SymmetryFactor            1.0
+        End
+    
+        Frequencies[1/cm]             {n_freq1}
+            {freq fragment 1}
+
+
+        ElectronicLevels[1/cm]      1
+            0    3
+
+        ZeroEnergy[kcal/mol]        0.0
+    End ! end fragment
+
+    Fragment       {fr_n} ! {chemid fragment 2}
+    RRHO
+        Geometry[angstrom]            {n_atoms2}
+        {geom fragment 2}
+
+        Core   RigidRotor
+            SymmetryFactor            1.0
+        End
+    
+        Frequencies[1/cm]             {n_freq2}
+        {freqs fragment 2}
+
+        ElectronicLevels[1/cm]      1
+            0    2
+
+        ZeroEnergy[kcal/mol]        0.0
+    End ! end fragment
+
+        GroundEnergy[kcal/mol]      {prod_energy}
+
+    
+    End ! end bimolecular
+!****************************************
```

Comment:
The user needs to fill in this file with everything except `{prod_energy}` and have it in their kinbot or  pes main directory.

Reply:
The template file can be found in `/tpl` but the user needs to copy the file into their main kinbot or pes run directory and edit everything in the file except for the `{prod_energy}` value.